jupyterlab_bigquery/src/components/details_panel/dataset_details_panel.tsx

```diff
@@ -49,7 +50,25 @@ export default class DatasetDetailsPanel extends React.Component<Props, State> {
       const details = await this.props.datasetDetailsService.listDatasetDetails(
         this.props.dataset_id
       );
-      this.setState({ hasLoaded: true, details });
+
```

Comment:
Here and in `table_details_panel.tsx` I pull out the details that need to be shown in the striped table format and store them in `rows` to be passed to the common `details_panel.tsx`